projects/ssr-tests/src/ssr-testing.spec.ts

```diff
@@ -1,54 +1,126 @@
-import * as Log from './log.utils';
-import * as ProxyServer from './proxy.utils';
-import * as Ssr from './ssr.utils';
+import { Server } from 'http';
+import { expectLogMessages } from './matchers/matchers';
+import * as LogUtils from './utils/log.utils';
+import * as ProxyUtils from './utils/proxy.utils';
+import * as SsrUtils from './utils/ssr.utils';
 
 const BACKEND_BASE_URL: string = process.env.CX_BASE_URL || '';
 
 describe('SSR E2E', () => {
-  let proxy: any;
+  let proxy: Server;
```

Comment:
[MINOR]
To avoid ambiguity what we're proxying (ssr server vs backend), what about a rename
`proxy` -> `backendProxy`
?